code/api/src/modules/user/model.js

```diff
@@ -36,6 +37,10 @@ module.exports = function(sequelize, DataTypes) {
     description: {
       type: DataTypes.STRING
     },
+    deliveryDate: {
+      type: DataTypes.STRING,
+      defaultValue: '1st'
```

Comment:
This makes it so if no value is given, the default value when creating a user is to have their delivery date set to the 1st every month.

Reply:
Great, that's how it'll render on our end